inspector/inspector-task.ts

```diff
@@ -71,3 +74,172 @@ task("inspect", "Main task")
+      }
+
+      let blocks = "";
+      blockRecords
+        .map((block: bigint) => Number(block))
+        .sort((a, b) => a - b)
+        .map((block) => (blocks = blocks + block + " "));
+
+      console.log(
+        `Number of blocks/seconds it took to fulfill ${count} requests, in ascending order: `
+      );
+      console.log(blocks);
+
+      await loadTestVRFConsumer.clear();
+    } catch (e) {
+      console.error(e);
+    }
+  });
+
+task("load-test-rr", "Load test task")
+  .addOptionalParam("batch", "Batch size")
+  .setAction(async (taskArgs, hre: HardhatRuntimeEnvironment) => {
+    try {
+      const { ethers, deployments, network } = hre;
+      const batch = Number(taskArgs.batch) || 1;
+
+      const accId =
+        network.name == "baobab"
+          ? (process.env.BAOBAB_ACC_ID as string)
+          : (process.env.CYPRESS_ACC_ID as string);
+
+      const _loadTestRRConsumer = await deployments.get("LoadTestRRConsumer");
+      const loadTestRRConsumer = await ethers.getContractAt(
+        _loadTestRRConsumer.abi,
+        _loadTestRRConsumer.address
+      );
+
+      const callbackGasLimit = 500_000;
+
+      console.log(
+        `Batch size: ${batch}. Each batch contains 50 requests. Total requests: ${
+          50 * batch
+        }`
+      );
+      for (let i = 0; i < batch; i++) {
+        await (
+          await loadTestRRConsumer.requestRR(accId, callbackGasLimit)
+        ).wait();
+      }
+
+      const count = 50 * batch;
+      let len = 0;
+
+      while (len < count) {
+        len = await loadTestRRConsumer.getBlockRecordsLength();
+      }
+
+      const blockRecords = [];
+      for (let i = 0; i < count; i++) {
+        blockRecords.push(await loadTestRRConsumer.blockRecords(i));
+      }
+
+      let blocks = "";
+      blockRecords
+        .map((block: bigint) => Number(block))
+        .sort((a, b) => a - b)
+        .map((block) => (blocks = blocks + block + " "));
+
+      console.log(
+        `Number of blocks/seconds it took to fulfill ${count} requests, in ascending order: `
+      );
+      console.log(blocks);
+
+      await loadTestRRConsumer.clear();
+    } catch (e) {
+      console.error(e);
+    }
+  });
+
+task("addConsumer", "Add consumer")
+  .addOptionalParam("consumer", "Consumer Contract Name")
+  .setAction(async (taskArgs, hre) => {
+    const { deployments } = hre;
+    const accId =
+      network.name === "baobab"
+        ? process.env.BAOBAB_ACC_ID
+        : process.env.CYPRESS_ACC_ID;
+
+    // consumer options are: "inspector", "vrf", "rr"
+    let consumerContractName;
+    switch (taskArgs.consumer) {
+      case "inspector":
+        consumerContractName = "InspectorConsumer";
+        break;
+      case "vrf":
+        consumerContractName = "LoadTestVRFConsumer";
+        break;
+      case "rr":
+        consumerContractName = "LoadTestRRConsumer";
+        break;
+      default:
+        consumerContractName = "InspectorConsumer";
```

Comment:
as I suggested before, it'd be nice if we don't have to deploy multiple contracts to run inspector scripts